src/components/CommentItem/CommentItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { useContext } from 'react';
+import { Comment } from '../../types/Comment';
+import * as commentAPI from '../../api/commentAPI';
+import { PostContext } from '../../context/PostContext';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentItem: React.FC<Props> = ({ comment }) => {
+  const { setComments, setErrorMessage } = useContext(PostContext);
+
+  const deleteComment = () => {
+    setComments(currentComments => currentComments
```

Comment:
why do you need to write logic of removing item here in this component? write function to remove it in your context and use just 1 function here, pass only comment id to remove